src/runtime/pkg/katautils/config.go

```diff
@@ -74,74 +74,78 @@ type factory struct {
+	DiskRateLimiterBwMaxRate       uint64   `toml:"disk_rate_limiter_bw_max_rate"`
+	DiskRateLimiterBwOneTimeBurst  uint64   `toml:"disk_rate_limiter_bw_one_time_burst"`
+	DiskRateLimiterOpsMaxRate      uint64   `toml:"disk_rate_limiter_ops_max_rate"`
+	DiskRateLimiterOpsOneTimeBurst uint64   `toml:"disk_rate_limiter_ops_one_time_burst"`
+	RxRateLimiterMaxRate           uint64   `toml:"rx_rate_limiter_max_rate"`
+	TxRateLimiterMaxRate           uint64   `toml:"tx_rate_limiter_max_rate"`
+	NetRateLimiterBwMaxRate        uint64   `toml:"net_rate_limiter_bw_max_rate"`
+	NetRateLimiterBwOneTimeBurst   uint64   `toml:"net_rate_limiter_bw_one_time_burst"`
+	NetRateLimiterOpsMaxRate       uint64   `toml:"net_rate_limiter_ops_max_rate"`
+	NetRateLimiterOpsOneTimeBurst  uint64   `toml:"net_rate_limiter_ops_one_time_burst"`
+	MemOffset                      uint64   `toml:"memory_offset"`
+	VirtioFSCacheSize              uint32   `toml:"virtio_fs_cache_size"`
+	DefaultMaxVCPUs                uint32   `toml:"default_maxvcpus"`
+	MemorySize                     uint32   `toml:"default_memory"`
+	MemSlots                       uint32   `toml:"memory_slots"`
+	DefaultBridges                 uint32   `toml:"default_bridges"`
+	Msize9p                        uint32   `toml:"msize_9p"`
+	PCIeRootPort                   uint32   `toml:"pcie_root_port"`
+	NumVCPUs                       int32    `toml:"default_vcpus"`
+	BlockDeviceCacheSet            bool     `toml:"block_device_cache_set"`
+	BlockDeviceCacheDirect         bool     `toml:"block_device_cache_direct"`
+	BlockDeviceCacheNoflush        bool     `toml:"block_device_cache_noflush"`
+	EnableVhostUserStore           bool     `toml:"enable_vhost_user_store"`
+	DisableBlockDeviceUse          bool     `toml:"disable_block_device_use"`
+	MemPrealloc                    bool     `toml:"enable_mem_prealloc"`
+	HugePages                      bool     `toml:"enable_hugepages"`
+	VirtioMem                      bool     `toml:"enable_virtio_mem"`
+	IOMMU                          bool     `toml:"enable_iommu"`
+	IOMMUPlatform                  bool     `toml:"enable_iommu_platform"`
+	Debug                          bool     `toml:"enable_debug"`
+	DisableNestingChecks           bool     `toml:"disable_nesting_checks"`
+	EnableIOThreads                bool     `toml:"enable_iothreads"`
+	DisableImageNvdimm             bool     `toml:"disable_image_nvdimm"`
+	HotplugVFIOOnRootBus           bool     `toml:"hotplug_vfio_on_root_bus"`
+	DisableVhostNet                bool     `toml:"disable_vhost_net"`
+	GuestMemoryDumpPaging          bool     `toml:"guest_memory_dump_paging"`
+	ConfidentialGuest              bool     `toml:"confidential_guest"`
+	GuestSwap                      bool     `toml:"enable_guest_swap"`
+	Rootless                       bool     `toml:"rootless"`
+	DisableSeccomp                 bool     `toml:"disable_seccomp"`
+	DisableSeLinux                 bool     `toml:"disable_selinux"`
```

Comment:
On a personal note, these huge chunks are part of the reason I've learnt to dislike spurious formatting in source code.  For an uninitiated, it takes a bit of staring into this to realise that the whole thing is (probably) just about adding a couple of lines.  Anyway, just saying. ;-)